client/my-sites/pages/page/index.js

```diff
@@ -521,34 +523,45 @@ class Page extends Component {
 			<div className={ classNames( hierarchyIndentClasses ) } />
 		);
 
+		const innerPageTitle = (
+			<>
+				{ depthIndicator }
+				{ title }
+				{ ! isTrashed && latestPostsPage && (
```

Comment:
Is the `! isTrashed` condition needed? Seems we don't display the popover on trashed pages anyway:

https://github.com/Automattic/wp-calypso/blob/4d10d4a6a9d6fae73ee17c96f35908c095cd598f/client/my-sites/pages/page/index.js#L545